src/archivematicaCommon/lib/archivematicaFunctions.py

```diff
@@ -75,9 +75,24 @@ def unicodeToStr(string):
         string = string.encode("utf-8")
     return string
 
-def strToUnicode(string):
+def strToUnicode(string, obstinate=False):
     if isinstance(string, str):
-        string = string.decode("utf-8")
+        if obstinate:
+            # Obstinately try to get a Unicode instance: attempt to decode to
+            # using the 'ascii', 'utf8', and finally 'latin-1' codecs. If all
+            # fail, decode using 'ascii', ignoring indecipherable bytes.
+            try:
+                string = string.decode('ascii')
+            except UnicodeDecodeError:
+                try:
+                    string = string.decode('utf8')
+                except UnicodeDecodeError:
+                    try:
+                        string = string.decode('latin-1')
+                    except UnicodeDecodeError:
+                        string = string.decode('ascii', 'ignore')
```

Comment:
Looks good!

I was thinking about this more, and I think it might make more sense to immediately fall back to `string.decode('utf8', 'replace')`

Eg
```
>>> sj = u'元気だ'.encode('shiftjis')
>>> sj
'\x8c\xb3\x8bC\x82\xbe'
>>> print sj.decode('latin1')
³C¾
>>> print sj.decode('utf8', 'ignore') 
C
>>> print sj.decode('utf8', 'replace')
���C��
```

They're all nonsense, but utf8 with ignore makes it clear what happened.